src/content/peerconnection/multiple/js/main.js

```diff
@@ -94,10 +111,10 @@ function onCreateSessionDescriptionError(error) {
   console.log(`Failed to create session description: ${error.toString()}`);
 }
 
-function gotDescription1Local(desc) {
+async function gotDescription1Local(desc) {
   pc1Local.setLocalDescription(desc);
   console.log(`Offer from pc1Local\n${desc.sdp}`);
-  pc1Remote.setRemoteDescription(desc);
+  await pc1Remote.setRemoteDescription(desc);
```

Comment:
this one made me scratch my head... since sCP was called too late.
This sample needs to be brought up to modern standards but do we have a spec problem here?